### PR TITLE
Update JSON and JEC files

### PR DESCRIPTION
**Pull request by aperloff**
This PR will update the data JSON files and the JEC conditions. There is no update to the JER conditions right now.

### CODE REVIEW

File: Production/python/scenarios.py [hunk 1]
```diff
@@ -60,6 +60,30 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "Fall17":
+            self.set_vars(globaltag="94X_mc2017_realistic_v13",
+                          tagname="PAT",
+                          geninfo=True,
+                          jecfile="data/jec/Fall17_17Nov2017_V6_MC",
+                          jerfile="data/jer/Spring16_25nsV6_MC",
+                          pufile="TreeMaker/Production/test/data/",
```
Comment (kpedro88) [inline Production/python/scenarios.py]: this is not a file

File: Production/python/scenarios.py [hunk 2]
```diff
@@ -60,6 +60,30 @@ def __init__(self,sname):
...
+        elif sname == "Fall17sig":
+            self.set_vars(globaltag="94X_mc2017_realistic_v13",
+                          tagname="PAT",
+                          geninfo=True,
+                          signal=True,
+                          jecfile="data/jec/Fall17_17Nov2017_V6_MC",
+                          jerfile="data/jer/Spring16_25nsV6_MC",
+                          pufile="TreeMaker/Production/test/data/",
+                          era="Run2_25ns")
+        elif sname == "EOY2017ReReco":
```
Comment (kpedro88) [inline Production/python/scenarios.py]: I would prefer to follow the convention from 2016, i.e. `2017ReReco17Nov`

File: Production/python/scenarios.py [hunk 3]
```diff
@@ -60,6 +60,30 @@ def __init__(self,sname):
                           jecfile="data/jec/Summer16_23Sep2016AllV3_DATA",
                           residual=True,
                           era="Run2_25ns")
+        elif sname == "Fall17":
```
Comment (kpedro88) [inline Production/python/scenarios.py:63]: Update README.md to list new scenarios

**Comment by aperloff** [comment]
@kpedro88 I have addressed all of your comments. Please review again.

**Pull request merged**